src/app/CASEClient.h

```diff
@@ -34,6 +35,7 @@ struct CASEClientInitParams
     Messaging::ExchangeManager * exchangeMgr                           = nullptr;
     FabricTable * fabricTable                                          = nullptr;
     Credentials::GroupDataProvider * groupDataProvider                 = nullptr;
+    Crypto::SessionKeystore * sessionKeystore                          = nullptr;
```

Comment:
Since SessionManager is the canonical place for the SessionKeystore, perhaps every class that has a reference to SessionManager (e.g. CASE/PASE classes) could always use a getter on SessionManager to access. Not a big deal and can keep like this, but it may have simplified. Just not sure if there would be other concerns related to this.

Reply:
That's actually a good idea. It can simplify things a bit.